_layouts/news.html

```diff
@@ -19,7 +19,48 @@
  {% assign coverimage = "/assets/images/GTN.png" %}
  {% assign coverimagealt = "GTN logo with a multi-coloured star and the words Galaxy Training Network" %}
 {% endif %}
-
+<script type="application/ld+json">
+    {
+      "@context": "https://schema.org",
+      "@type": "BlogPosting",
+      "url": "{{ site.url }}{{ page.url }}",
+      "name": {{ page.title | jsonify }},
+      "headline": {{ page.title | jsonify }},
+      "keywords": {{ page.tags | join: ',' | jsonify }},
+      "description": {{ page.excerpt | strip_newlines | strip | jsonify }},
+      "articleBody": {{ page.content | strip_html | jsonify }},
+      "datePublished": {{ page.date | jsonify }},
+      "dateModified": {{ page.last_modified_at | default: page.date | jsonify }},
+      "author": {
+        "@type": "Person",
+        "name": {{ site.author_name | jsonify }},
+        "givenName": {{ site.author_first_name | jsonify }},
+        "familyName": {{ site.author_last_name | jsonify }},
+        "email": {{ site.email | jsonify }}
+      },
+      "publisher": {
+        "@type": "Organization",
+        "name": {{ site.title | jsonify }},
+        "url": "{{ site.url }}",
+        "logo": {
+          "@type": "ImageObject",
+          "width": 32,
+          "height": 32,
+          "url": "{{ site.url }}/icon/favicon.ico"
```

Comment:
```suggestion
          "url": "{{ site.url }}{{ site.baseurl }}{{ site.small_logo }}"
```

that logo is 60x60 px